101-event-grid-servicebus-topic/azuredeploy.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
add defaultValues for the parameters where appropriate - be sure to use unique values where required (and update the metadata descriptions)

Reply:
How can I ensure that the defaultValues are unique? GEN-UNIQUE works only in the parameter file.